src/adapters/adbund.js

```diff
@@ -0,0 +1,81 @@
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+
+var adBundAdapter = function adBundAdapter() {
+  var timezone = (new Date()).getTimezoneOffset();
+  var bidAPIs = [
+    'http://us-east-engine.adbund.xyz/prebid/ad/get',
+    'http://us-west-engine.adbund.xyz/prebid/ad/get'
+  ];
+  //根据时区来选择接口服务器
+  var bidAPI = bidAPIs[timezone < 0 ? 0 : 1];
+  //bidAPI = 'http://52.66.158.121:8888/prebid/ad/get';
+
+  function _stringify(param) {
+    var result = [];
+    var key;
+    for (key in param) {
+      if (param.hasOwnProperty(key)) {
+        result.push(key + '=' + encodeURIComponent(param[key]));
+      }
+    }
+    return result.join('&');
+  }
+
+  function _jsonp(server, param, handler) {
+    var head = document.getElementsByTagName('head')[0];
+    var script = document.createElement('script');
+    var callbackName = 'jsonp_' + (new Date()).getTime().toString(36);
+
+    param[param.jsonp] = callbackName;
+    window[callbackName] = function (data) {
+      if (typeof handler === 'function') {
+        handler(data);
+      }
+      try {
+        window[callbackName] = undefined;
+        script.parentNode.removeChild(script);
+      } catch (e) {}
+    };
+
+    script.charset = 'utf-8';
+    script.src = server + '?' + _stringify(param);
+    head.insertBefore(script, head.lastChild);
+  }
+
+  function _requestBids(bid) {
+    var info = {
+      referrer: utils.getTopWindowUrl(),
+      domain: utils.getTopWindowLocation().hostname,
+      ua: window.navigator.userAgent
+    };
+    var param = Object.assign({}, bid.params, info);
+    param.sizes = JSON.stringify(param.sizes || bid.sizes);
+    param.jsonp = 'callback';
+    _jsonp(bidAPI, param, function (data) {
+      var response;
+      if (data && data.cpm) {
+        response = bidfactory.createBid(1);
```

Comment:
It is recommended to use CONSTANTS.STATUS.GOOD as defined in [constants.json](https://github.com/prebid/Prebid.js/blob/master/src/constants.json)